spec

```diff
@@ -10466,5 +10466,6 @@ endseg
 beginseg
     name "softsprite_matrix_static"
     romalign 0x1000
-    include "build/baserom/softsprite_matrix_static.o"
+    include "build/src/code/softsprite_matrix_static.o"
```

Comment:
probably shouldn't be code but no clue where tbh
there are more of these so maybe a folder `src/segments/` ? no clue

Reply:
Yeah, this definitely shouldn't be in `code/`. 
I like the  `segments/` idea